src/CoreImage/CIImageInitializationOptions.cs

```diff
@@ -34,53 +34,20 @@
 
 namespace XamCore.CoreImage {
 
-	public class CIImageInitializationOptions : DictionaryContainer
+	public partial class CIImageInitializationOptions
 	{
 #if !COREBUILD
-		public CIImageInitializationOptions ()
-			: base (new NSMutableDictionary ())
-		{
-		}
-
-		public CIImageInitializationOptions (NSDictionary dictionary)
-			: base (dictionary)
-		{
-		}
-
 		public CGColorSpace ColorSpace {
 			get {
-				return GetNativeValue<CGColorSpace> (CIImage.CIImageColorSpaceKey);
+				return GetNativeValue<CGColorSpace> (CIImageInitializationOptionsKeys.ColorSpaceKey);
 			}
 			set {
-				SetNativeValue (CIImage.CIImageColorSpaceKey, value == null ? null : value);
+				SetNativeValue (CIImageInitializationOptionsKeys.ColorSpaceKey, value == null ? null : value);
 			}
 		}
 #endif
 	}
 
-	public class CIImageInitializationOptionsWithMetadata : CIImageInitializationOptions
-	{
-#if !COREBUILD
-		public CIImageInitializationOptionsWithMetadata ()
-		{
-		}
-
-		public CIImageInitializationOptionsWithMetadata (NSDictionary dictionary)
```

Comment:
According to api-diff, this constructor is gone:

> public CIImageInitializationOptionsWithMetadata (Foundation.NSDictionary dictionary);

Reply:
Yea I had `CIImageInitializationOptionsWithMetadata` be generated at some point but moved away from that and forgot to re-add the .ctor.

API diff reports other breaking changes though:

<img width="579" alt="screenshot 2017-11-17 11 15 01" src="https://user-images.githubusercontent.com/7839202/32957169-e938a2bc-cb88-11e7-8dbe-566716304da3.png">

However according to https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md this should be fine...

Reply:
api-diff will incorrectly report API that moved to a base class as a breaking change.

This is a known issue with the API diff.